README.md

```diff
@@ -59,18 +59,31 @@ you the tables and ActiveRecord models, and also an engine with the necessary AP
 
 Now is your turn. Before proceeding, you need to set these ENV variables:
 * `PASSKIT_WEB_SERVICE_HOST`
-* `PASSKIT_CERTIFICATE_KEY`
-* `PASSKIT_PRIVATE_P12_CERTIFICATE`
 * `PASSKIT_APPLE_INTERMEDIATE_CERTIFICATE`
 * `PASSKIT_APPLE_TEAM_IDENTIFIER`
 * `PASSKIT_PASS_TYPE_IDENTIFIER`
 
 We have a [specific guide on how to get all these](docs/passkit_environment_variables.md), please follow it.
 You cannot start using this library without these variables set, and we cannot do the work for you.
 
+### Access to certificate data
+
+Use the database or environment variables to access certificate data.
+By default, the certificate and its password will be sourced from the database.
+If you want to use environment variables, set:
+* `PASSKIT_CERTIFICATE_KEY`
+* `PASSKIT_CERTIFICATE_PASSWORD`
+
+and update the configuration as follows:
+```ruby
+Passkit.configure do |config|
+  config.use_database_for_certificates = false
```

Comment:
This configuration is not needed if we make the gem accept `certificate_source` object and call `#certificate` and `#password` methods on it.

Reply:
without this configuration, ENV variables will be checked each time

Reply:
Why check ENV variables when certificate_source is passed?